phac-nml/irida-next
# Update sidebar & breadcrumbs when editing project

## What does this PR do and why?
Updates the project name in the layout sidebar and breadcrumbs when a project name is successfully updated.
Pulled from [#189](https://github.com/phac-nml/irida-next/pull/189).

## Screenshots or screen recordings
![Screenshot from 2023-09-21 16-14-36](https://github.com/phac-nml/irida-next/assets/325703/9e625a58-ec81-4048-bb91-3985a5ee3ec6)

## How to set up and validate locally
1. Navigate to a Project
2. Click the `Settings` link in the sidebar
3. In the `Project name and description` form set `Project Name` to `a`
4. Click `Update project`
5. See that the form displays an error message and the sidebar and breadcrumbs are not updated.
6. Change the `Project Name` to `New project name`.
7. Click `Update project`
8. See a success message and that the sidebar an breadcrumbs have changed to the new name.

Note: Verify that change path refreshes the page to the new URL as expected.

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.
